### DISCUSSION TITLE
This is the best client I've found (and I've tried them all)

### DISCUSSION THREAD

**Discussion by volcs0** [tamland/airsonic-refix#48] [General]
Just wanted to say that this has been the best client for Airsonic that I've found. I'm actually running Navidrome on my unRAID server. I installed a Docker container for airsonic-refix, fixed the URL, and everything runs so fast and smoothly. This is the closest to a "Spotify-like experience" that I've seen. Congrats on a great interface.

**Comment by kiike** [comment]
I agree! Definitely a mandatory Airsonic fix. I spend all day in Airsonic and this is very much welcome. Good job @tamland!

**Comment by Berg-HK** [comment]
Great work and thank you for your contribution. This is a great alternative GUI.

**Comment by JPGuillemin** [comment]
Should try Airdrome :)
